EKF/covariance.cpp

```diff
@@ -172,7 +172,7 @@ void Ekf::predictCovariance()
 	float wind_vel_sig;
 
 	// Don't continue to grow wind velocity state variances if they are becoming too large or we are not using wind velocity states as this can make the covariance matrix badly conditioned
```

Comment:
@priseborough What do you think, should we increase wind variances even if our wind flag is set to false? Will it lead to a badly conditioned matrix?